canvas_sdk/effects/launch_modal.py

```diff
@@ -22,11 +22,17 @@ class TargetType(StrEnum):
     url: str | None = None
     content: str | None = None
     target: TargetType = TargetType.DEFAULT_MODAL
+    title: str = "Modal"
```

Comment:
Added two alternate suggestions, feel free to choose either or propose something else. I think "Modal" is too technical, and this is something that is user-facing.